addons/web_editor/static/src/js/editor/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -1390,6 +1391,368 @@ describe('Paste', () => {
                     contentAfter: '<ul><li>123</li><li>456[]abc</li><li>def</li><li>ghi</li></ul>',
                 });
             });
+            it('should paste a nested list into another list', async () => {
+                await testEditor(BasicEditor, {
+                    contentBefore: '<ol><li>Alpha</li><li>Beta</li><li>[]</li></ol>',
+                    stepFunction: async editor => {
+                        await pasteHtml(editor, unformat(`
+                            <ul>
+                                <li>abc</li>
+                                <li>def</li>
+                                <li class="oe-nested">
+                                    <ul>
+                                        <li>123</li>
+                                        <li>456</li>
+                                    </ul>
+                                </li>
+                            </ul>
+                        `));
+                    },
+                    contentAfter: unformat(`
+                        <ol>
+                            <li>Alpha</li>
+                            <li>Beta</li>
+                            <li>abc</li>
+                            <li>def</li>
+                            <li class="oe-nested">
+                                <ol>
+                                    <li>123</li>
+                                    <li>456[]</li>
+                                </ol>
+                            </li>
+                        </ol>
+                    `),
+                });
+            });
+            it('should paste a nested list into another list', async () => {
+                //TODO: change this test case after fixing copy paste in empty node (task-3302332)
```

Comment:
You can remove this comment, whoever handles that task will change this if needed.

Reply:
Sorry for the unnecessary comments in the testcases the other task mentioned here was green in Testing IN when this task was in dev with me, so we thought it would be merged before this and we'll rebase it and remove this comments, however the other task was not merged before this. I'll remove the comments from the testcases.